code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -11,7 +11,7 @@
 /obj/effect/mob_spawn/human/corpse/syndicatesoldier
 	name = "Syndicate Operative"
 	uniform = /obj/item/clothing/under/syndicate
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat

```

Comment:
nah but it looks pretty similar anyways the sprites are out of date anyways I can if its really critical